src/pyFAI/gui/pilx/MainWindow.py

```diff
@@ -141,6 +142,41 @@ def initData(self,
             else:
                 self._offset = 0
 
+            # Find source dataset paths
+            cnt = 0
+            for char in dataset_path[-1::-1]:
```

Comment:
Not sure the first `-1` is needed (no need to remove the first character)

Reply:
Actually it is the same, at least since python 2.5. I did not update my knowledge base for a while.

Reply:
Indeed :+1: